crates/sui-types/src/unit_tests/messages_tests.rs

```diff
@@ -270,7 +282,11 @@ fn test_auth_sig_commit_to_wrong_epoch_id_fail() {
     {
         let (_, sec): (_, AuthorityKeyPair) = get_key_pair();
         // signature commits to epoch 0
-        let sig = AuthoritySignature::new(&message, 1, &sec);
+        let sig = AuthoritySignature::new_secure(
+            &IntentMessage::new(Intent::default(), message.clone()),
+            &1,
```

Comment:
what's this 1? comment says it commits to zero

Reply:
this is a red herring test as you see this is in a dropped scope... modified to capture the pos and negative case for committing to the epoch value.